### PR TITLE
Get the archivematica-infra TF to plan/apply cleanly in terraform0.14

### PR DESCRIPTION
**Pull request by alexwlchan**
This is the version of TF that got installed on my new machine, so here we are. ✨

### CODE REVIEW

File: terraform/stack_staging/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/hashicorp/aws" {
```
Comment (kenoir) [inline terraform/stack_staging/.terraform.lock.hcl:4]: Guessing this is a new thing in the most recent version of terraform?

**Review by kenoir — APPROVED** [review]

File: terraform/stack_staging/.terraform.lock.hcl (hunk 1 above)
Comment (alexwlchan) [reply terraform/stack_staging/.terraform.lock.hcl:4]: Yup. https://www.terraform.io/docs/configuration/dependency-lock.html

tl;dr is that changes to your dependencies are now explicitly visible (as opposed to the version hints we used to set in `provider.tf`), so you can discuss when a dependency changes.
